system-tests/e2e-suite/src/test/scala/com/hotels/styx/proxy/ChunkedDownloadSpec.scala

```diff
@@ -44,12 +46,14 @@ class ChunkedDownloadSpec extends FunSpec
   with TestClientSupport
   with Eventually {
 
-  val (originOne, originOneServer) = originAndCustomResponseWebServer("NettyOrigin")
+  val (originOne, originOneServer) = originAndCustomResponseWebServer("NettyOrigin-01")
```

Comment:
The solution is fine but I would expect that we just create the origin in  each test / setup method if they can't be reused?

Reply:
It is the styx server that would have to be created in each test/setup cycle.

Reply:
That's also possible, but I was just thinking of limiting the scope of the origins ( they can be created with ,e.g., a random name in each test). But since , as you say, there's only one Styx instance it makes sense to set all of them up in the fixture.